doc/authoring_command_modules/README.md

```diff
@@ -199,28 +199,39 @@ azdev style --module <module> [--pylint] [--pep8]
 Submitting Pull Requests
 ------------------------
 
-### Modify Change Log
+### Format PR Title
 
-Modify the `HISTORY.rst` for any customer-facing changes. If a module has changed at all since a previous release so that a version bump is required, it is fine to add a generic entry that says "* Minor fixes.".
+History notes are auto-generated based on commit messages and descriptions starting from [S165](https://github.com/Azure/azure-cli/milestone/82). The commmit message for the squashed merge commit defaults to the PR title. Starting from 01/30/2020, we require all the PR titles to follow the below format:
+1. [**Mandatory**] Each commit message must start with `[Component Name]` or `{Component Name}`. 
+    * `Component Name` shall be replaced by the real ones such as `Storage`, `Compute`. It could be the name of a command module, but in title case with necessary spaces for better readability, such as `API Management`, `Managed Service`. Other possible component names include but are not limited to: `Packaging`, `Misc.`, `Aladdin`.
```

Comment:
Do we have a list of valid component names?

Reply:
Based on statistics of all past history notes, most component names can be mapped to command module name, the format could vary though. So we just want to use the same format for component name. 

I don't think we want to limit the name to a list as we will have new modules and also other aspects like changes for CI, Packaging and more. We may come up with component name recommendations for changes not related to command modules when we have more data later.